tasks/release.py

```diff
@@ -59,8 +58,12 @@ def update_changelog(ctx, new_version):
 
     # removing releasenotes from bugfix on the old minor.
     previous_minor = "%s.%s" % (new_version_int[0], new_version_int[1] - 1)
-    ctx.run("git rm --ignore-unmatch `git log {}.0...remotes/origin/{}.x --name-only \
-            | grep releasenotes/notes/`".format(previous_minor, previous_minor))
+    try:
+        log_result = ctx.run("git log {}.0...remotes/origin/{}.x --name-only | \
+                grep releasenotes/notes/".format(previous_minor, previous_minor))
+        ctx.run("git rm --ignore-unmatch {}".format(log_result.stdin))
```

Comment:
Shouldn't this be `log_result.stdout`?

Reply:
you're 100% right, this is wrong, it should be `stdout`